gradle/libraries.gradle

```diff
@@ -100,6 +100,8 @@ ext {
         springframework_restdocs:              "org.springframework.restdocs:spring-restdocs-mockmvc",
         springframework_security_test:         "org.springframework.security:spring-security-test", 
         springframework_web:                   "org.springframework:spring-web",
+		
+		micrometer_prometheus:                 "io.micrometer:micrometer-registry-prometheus",
```

Comment:
Please have a look if only github displays this incorrectly or if there is a problem with indentation.